guides/client/js-interop.md

```diff
@@ -223,6 +223,9 @@ like this:
 Then a hook callback object could be defined and passed to the socket:
 
 ```javascript
+/**
+ * @type {Object.<string, import("phoenix_live_view").ViewHook>}
```

Comment:
Yes, this is a valid JS code.

Reply:
`Object.<string, ViewHook>` is the Google Closure syntax which is a valid syntax in JSDoc.  In typescript it would be `{[key: string]: ViewHook}`.  Both syntaxes are understood by JSDoc and TypeScript.